### PR TITLE
UI Enhancements

### PR DESCRIPTION
**Pull request by esitarz**
#102: Scale Chakra-UI Theme Structure
#103: Revise Color Palette
#104: Ensure proper usage of Chakra throughout components _[incomplete]_
#105: Update branding
#106: Audit components for dark mode _[incomplete]_
#107: Audit container spacing _[incomplete]_
#108: Refactor TopNav and SideNav
#109: Refactor Dashboard

#110: Product List UI Enhancements:
-- toolbar
-- product card

#111: Theming Drawer: with color customization _[incomplete]_

Adjust several settings in the workspace linter.
Minor package updates to outdated dependencies.

### CODE REVIEW

File: src/components/shared/DebouncedSearchInput/DebouncedSearchInput.tsx [hunk 1]
```diff
@@ -36,9 +36,10 @@ const DebouncedSearchInput: FC<IDebouncedSearchInputProps> = ({
   }, [])
 
   return (
-    <InputGroup colorScheme="brand" maxW="300px">
-      <Input
+    <InputGroup colorScheme="brand" maxW={"sm"}>
+      <Input size="sm"
```
Comment (robertsoniv) [inline src/components/shared/DebouncedSearchInput/DebouncedSearchInput.tsx:40]: Let's keep inputs and buttons the default size everywhere

**Review by robertsoniv — CHANGES_REQUESTED** [review]
See my comments on the vercel deployment as well! Also it seems like you're using a different formatting so a lot mote is "changed" in some files than necessary.

**Comment by esitarz** [comment]
> See my comments on the vercel deployment as well!

I'll need you to resolve those or will need team access
image

> Also it seems like you're using a different formatting so a lot mote is "changed" in some files than necessary.

check out this [commit](https://github.com/Sitecore/Sitecore.Commerce.Headstart.ReactAdmin/pull/112/commits/1a0156ed19697a6fc5b0406deb0fe84807892ca8). My whole file was lighting up red because of a Prettier linting issue-- surprised more people don't see that. I think you posted a possible solution in the Teams chat? but it didn't seem related to the formatting changes I made in the workspace settings.

**Review by robertsoniv — APPROVED** [review]

**Comment by esitarz** [comment]
test comment

**Pull request merged**